src/css/custom.css

```diff
@@ -177,15 +177,23 @@ h1.docTitle_node_modules-\@docusaurus-theme-classic-src-theme-DocItem- {
 
 .header-github-link:before {
     content: '';
-    width: 20px;
-    height: 20px;
+    width: 24px;
+    height: 24px;
     display: flex;
-    background: url('/img/github.svg') no-repeat;
```

Comment:
This is the official Github svg , commonly used by all docusaurus sites to implement the icon and is a compressed one,

Reply:
You have not answered the questions.

Reply:
No i dont see any advantage using this svg!

Reply:
Then why did you do this?

Reply:
I felt this could be a easier approach to use the external svg to link a background image , apologize for that , i can rebase to use the exisiting svg file!

Reply:
This is a design related issue, and if we use an external links for content then we have no control over elements of the design. Please resolve this.

Reply:
thanks for the feedback, I understood your feedback, resolving soon.

Reply:
@palisadoes need to create one more svg to have a darker one for the light mode, i hope that is fine, rest everything looks fine now